.clang-tidy

```diff
@@ -84,10 +84,10 @@ WarningsAsErrors: '*'
 #   - readability-use-anyofallof                                We prefer `for (auto item : items)` over `std::ranges::all_of()` with a lambda.
 #   - misc-no-recursion                                         Ignore general recommendation to avoid recursion, which we commonly use when working with query plans.
 #   - bugprone-easily-swappable-parameters                      Ignore issues with swappable parameters as we found them to be of no help.
-#   - bugprone-suspicious-include                               Unity builds use `#include`s for cpp files. However, unity builds significantly improve the runtime of clang-tidy,
-#                                                                which is otherwise the slowest step of the CI pipeline by a factor of two. Furthermore, the linter checks for 
-#                                                                these includes as well.
-#   - bugprone-unchecked-optional-access                        Too many false positives (with clang-tidy 14/15). TODO(anyone): Re-evaluate for future clang versions.
+#   - bugprone-suspicious-include                               Unity builds use `#include`s for cpp files. However, unity builds significantly improve compile times, which is
+#                                                                otherwise the slowest step of the CI pipeline by a factor of two. Furthermore, the linter checks for these
```

Comment:
If we want to have line breaks in this file, we should limit the lines to 120 characters.
Currently, the line breaks added by GitHub or my editor compete with the manual ones which makes this file a pain to read.